### DISCUSSION TITLE
Checklist cheatsheet during annotation

### DISCUSSION THREAD

**Discussion by ghost** [tosdr/edit.tosdr.org#1005] [Ideas]
It would be nice if I could see the checklist during annotating the terms of services. Then the items would get ticked as I go through the annotation process. I keep thinking I've answered all the questions only to find one I missed several days later.
I also find questions that the policies don't have answers for like "What happens to your data when they get acquired or when they shut down the service?" and "How do they work with government requests?"

**Comment by evilcel3ri** [comment]
Hi @kitcallie, there is a first draft of a checklist, you can find it here: https://github.com/tosdr/tosdr.org/wiki/checklist it isn't much and feel free to expand it if you want :)

**Comment by ghost** [comment]
I have that list bookmarked. I just have issues answering all of the questions on it when I'm annotating. That's why I was thinking after I answered one question it would no longer be available or would end up checked off a built-in list. Maybe just greyed out until approved then it would be unavailable?
For instance, I can't find which question hasn't been answered yet for Reddit, Tumblr, or Medium.
